src/build/arg/mod.rs

```diff
@@ -87,6 +89,8 @@ pub struct Arg<'help> {
     pub(crate) help_heading: Option<&'help str>,
     pub(crate) global: bool,
     pub(crate) exclusive: bool,
+    #[doc(hidden)]
+    pub value_hint: ValueHint,
```

Comment:
Please use `pub(crate)` instead of `doc(hidden)` and write a getter `get_value_hint()` so that you can use it in `clap_generate` pkg.